include/kernel.h

```diff
@@ -2863,6 +3031,7 @@ struct k_msgq {
 	char *read_ptr;
 	char *write_ptr;
 	u32_t used_msgs;
+	int flags;
```

Comment:
Not an issue with just one flag, but it's often a good idea to use unsigned integers for bitmasks.